lib/pi_piper/pin_error.rb

```diff
@@ -0,0 +1,3 @@
+module PiPiper
+  class PinError < StandardError; end
```

Comment:
True but I think number of files shouldn't be an issue. If we add any more error types we can instead put them into an `errors.rb` file.